README.md

```diff
@@ -56,15 +58,15 @@ Learn how to automate your workflow using Twilio's REST API and Twilio SMS. This
    pip install -r requirements.txt
    ```
 
-1. Edit the following keys/values for the `config.py` file inside the  `airtng_flask/` directory. Be sure to replace the place holders and connection string with real information. Replace the connection string preferably under development config.
+1. Copy the sample configuration `.env.example` to `.env`, and then edit `.env` to match your configuration.
 
-   ```
-   TWILIO_ACCOUNT_SID = 'your_twilio_account_sid'
-   TWILIO_AUTH_TOKEN = 'your_twilio_auth_token'
-   TWILIO_NUMBER = 'your_twilio_phone_number'
+  ```bash
+  cp .env.example .env
+  ```
 
-   SQLALCHEMY_DATABASE_URI = 'sqlite://'
-   ```
+  You can find your `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` in your
+   [Twilio Account Settings](https://www.twilio.com/console).
+   You will also need a `TWILIO_PHONE_NUMBER`, which you may find [here](https://www.twilio.com/console/phone-numbers/incoming).
```

Comment:
When i run this app and hit "Signup", nothing seems to happen. Am i missing another step in the process?